### PR TITLE
Fix issue when sprite texture was modified during animation

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

File: packages/dev/core/src/Sprites/spriteRenderer.ts [hunk 1]
```diff
@@ -10,10 +10,6 @@ import type { ISize } from "../Maths/math.size";
 
 import type { ThinTexture } from "../Materials/Textures/thinTexture";
 import type { Scene } from "../scene";
-
-import "../Engines/Extensions/engine.alpha";
```
Comment (deltakosh) [inline packages/dev/core/src/Sprites/spriteRenderer.ts:14]: @sebavan : I want to be sure this is not a problem for thinsprites

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15612/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15612/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15612/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15612/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15612/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15612/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to the build configuration file.

*This build will release a new package on npm*

If that was unintentional please make sure to revert those changes or close this PR.

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15612/merge/testResults/webgpuplaywright/index.html

**Review by sebavan — APPROVED** [review]

**Pull request merged**